docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       DBPASS: ${FLEXO_DB_PASS:-flexo}
       DBNAME: ${FLEXO_DB_NAME:-flexo}
       DBSSL: ${FLEXO_DB_SSL:-disable}
+      SECRET: ${SECRET:-shared_secret}
```

Comment:
```suggestion
      SECRET: ${FLEXO_SECRET:-shared_secret}
```

Reply:
this is to prevent collision with other env vars the user may have running when deploying via docker compose